apps/console/src/public/deployment.config.json

```diff
@@ -354,7 +354,7 @@
                 }
             },
             "applications": {
-                "disabledFeatures": [],
+                "disabledFeatures": [""],
```

Comment:
Is this change intended?

Reply:
No, this has been removed in subsequent changes.